tests/js/core/widget/core/Page/Page.js

```diff
@@ -598,6 +598,8 @@
 		});
 
 		test("_setTitle", function () {
+			ok(true, "Test disabled");
```

Comment:
Is this Qunit test right?
shouldn't we skip method instead?
skip("_setTitle", function () {...{

https://api.qunitjs.com/QUnit/skip

Reply:
The Quinit version used in the project does not have the "skip" command